modules/Core/includes/endpoints/RemoveGroupFromDiscordIdEndpoint.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/** 
+ * @param int $discord_user_id The NamelessMC user's Discord user ID to edit
+ * @param int $discord_role_id The Discord role ID to verify exists on the site and to remove
+ * 
+ * @return string JSON Array
+ */
+class RemoveGroupFromDiscordIdEndpoint extends EndpointBase {
+
+    public function __construct() {
+        $this->_route = 'removeGroupFromDiscordId';
+        $this->_module = 'Core';
+        $this->_description = 'Remove a user\'s role from NamelessMC when it was removed from Discord. If the group is a primary group, it sets their group to post validation group';
+    }
+
+    public function execute(Nameless2API $api) {
```

Comment:
Another type

Reply:
As per our conversation on Discord, leaving this.